### DISCUSSION TITLE
What ice modifier should I use for 'marlonrichert/zsh-autocomplete'?

### DISCUSSION THREAD

**Discussion by rasivasu** [zdharma-continuum/zinit#595] [Q&A] [answered]
Based on the documentation, I am using the following for 'zsh-users/zsh-autosuggestions'.
```zsh
zinit wait lucid atload'_zsh_autosuggest_start' light-mode for \
    zsh-users/zsh-autosuggestions
```

Likewise, I would like to know what is the equivalent of '_zsh_autosuggest_start' for 'marlonrchert/zsh-autocomplete'? I am not sure where to find various events like '_zsh_autosuggest_start'.  I don't find any guidance at 'marlonrchert/zsh-autocomplete' repo as well.

**Comment by vladdoster** [comment, marked as answer]
@rasivasu,

Here is my configuration:

> [!TIP]
> [Github link](https://github.com/vladdoster/dotfiles/blob/23852314cf226e34b158d7769ee0bdf2e64d0720/zsh/.config/zsh/rc.d/03-zinit.zsh#L121-L128) to relevant part in my zinit config.

```zsh
zinit wait light-mode lucid for \
  atinit"zicompinit; zicdreplay" \
    @zdharma-continuum/fast-syntax-highlighting \
  atload"_zsh_autosuggest_start" \
  atinit"bindkey '^_' autosuggest-execute;bindkey '^ ' autosuggest-accept;" \
    @zsh-users/zsh-autosuggestions \
  blockf atpull'zinit creinstall -q .' \
    @zsh-users/zsh-completions
```

> [!IMPORTANT]
> Load last and the order is important.

**Reply by rasivasu** [reply]
Thank you very much @vladdoster
